ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: true
+contact_links:
+- name: ":question: Question / Support"
+  url: https://github.com/orgs/ManageIQ/discussions
+  about: Please ask and answer questions here.
```

Comment:
The goal here was to push people asking questions to the discussions instead of opening actionable tickets.  It should show up at the bottom of the chooser list, but it's hard for me to know until we merge it.